---
layout: archive
title: "Software"
permalink: /software/
author_profile: true
---

I am the main developer for these packages:


__SIAMCAT__  
R package for Statistical Inference of Associations between Microbial
Communities And host phenoType  
[Github repository](https://github.com/zellerlab/siamcat)

__BAMBI/SIMBA__  
R packages for the simulation of microbiome and the evaluation of differential abundance testing methods  
[BAMBI Github repository](https://github.com/zellerlab/BAMBI) and [SIMBA Github repository](https://github.com/zellerlab/simba)



Additionally, I contributed to these packages:

__PhaVa__  
python package for identifying phase variable invertible regions in bacterial genomes using long-read sequencing  
[Github repository](https://github.com/patrickwest/PhaVa)  
_Developed by Patrick T West, see [this paper](https://www.nature.com/articles/s41586-024-07970-4)_  


__PHONEMeS__  
R package to model signalling networks based on untargeted phosphoproteomic data  
[Github repository](https://github.com/saezlab/PHONEMeS)  
_Originally developed by Camille DA Terfve, see [this publication](https://www.nature.com/articles/ncomms9033)_


__kinact__  
python Toolbox for Kinase Activity Scoring based on
phosphoproteomic data  
[Github repository](https://github.com/saezlab/kinact)  
_Now part of OmniPath package developed by Dénes Türei, see [the OmniPath Github repo](https://github.com/saezlab/omnipath)_
